src/librustdoc/html/markdown.rs

```diff
@@ -1259,14 +1261,15 @@ crate fn rust_code_blocks(md: &str, extra_info: &ExtraInfo<'_, '_>) -> Vec<RustC
                                 range: offset,
                                 code,
                                 syntax,
+                                is_ignore: lang_string.ignore != Ignore::None,
                             });
                             continue;
                         }
                     };
                     while let Some((Event::Text(_), offset)) = p.next() {
                         code_end = offset.end;
                     }
-                    (syntax, code_start, code_end, offset, true)
```

Comment:
You have `lang_string.ignore != Ignore::None` a lot - maybe make it a separate variable?